Adds some new tips related to lavaland/xenomorphs (#11711)
ParadiseSS13/Paradise: strings/tips.txt

As a Cargo Technician, you can hack MULEbots to make them faster, run over people in their way, and even let you ride them!
As a Cargo Technician, you can order contraband items from the supply shuttle console by de-constructing it and using a multitool on the circuit board, the re-assembling it.
As a Cargo Technician, you can earn more cargo points by shipping back crates from maintenance, liquid containers, plasma sheets, rare seeds from hydroponics, and more!

> As a Shaft Miner, northern part of lavaland has a lot more rare minerals than on the southern but is a lot more dangerous.

Reviewer: As a Shaft Miner, the northern part of Lavaland has a lot more rare minerals than the Southern part, but is a lot more dangerous.

> As a Shaft Miner, the northern part of Lavaland has a lot more rare minerals than the Southern part, but is a lot more dangerous.

Reviewer 2: Southern should be decapitalised

Reviewer 3: @improvedname please fix this so I can merge it.

As a Shaft Miner, always have a GPS on you, so a fellow miner or cyborg can come to save you if you die.

> As a Shaft Miner, you can use certain equipment to be safe from a ash storm, this can range from bluespace shelters to fireproof suits.

Reviewer: As a Shaft Miner, you can use certain equipment to be safe from ash storms. This equipment can range from bluespace shelters to fireproof suits.

> As a Shaft Miner, consider twice before you take out a tendril, some tendrils may be benifictal for your survival like the legion tendril for legion souls to heal yourself with.

Reviewer: As a Shaft Miner, think twice before taking out a tendril. Some tendrils may be beneficial to your survival. For instance, the Legion tendril can be used to farm Legion souls. These souls can be used to fully heal yourself.

> As a Ash Walker, you can refine ores with the bonfire

Reviewer: As an Ash Walker, you can refine ores using the bonfire in your nest.

As a Terror Spider, try putting webs on white floor tiles, they are much harder to see there.
As a Terror Spider, sometimes quickly pulling an enemy into a web can ensure victory in melee combat.
You can swap floor tiles by holding a crowbar in one hand and a stack of tiles in the other.

> When your restrained to alien nest and theres somebody in a nest next to you, you can unbuckle eachother but not yourself.

Reviewer 4: ```suggestion
When you are restrained to alien nest and there is somebody in a nest next to you, you can unbuckle each other but not yourself.
```

> When you are restrained to alien nest and there is somebody in a nest next to you, you can unbuckle each other but not yourself.

Reviewer: I don't think this tip should be added - this functionality is more so a bug than intentional behaviour and not something that should really be encouraged.
